packages/ingest/src/consumer.ts

```diff
@@ -42,10 +44,19 @@ export class Consumer {
   ): Promise<0 | 1> {
     try {
       await fn(this.queueUrl, message);
-      if (this.deleteProcessedMessage) await deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+      if (this.deleteProcessedMessage) {
+        await sqs.deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+      }
       return 1;
     } catch (error) {
-      log.error(error);
+      if (error.code === 'ExecutionAlreadyExists') {
+        log.debug('Deleting message for execution that already exists...');
+        await sqs.deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+        log.debug('Completed deleting message.');
```

Comment:
It seems we should return 1 here.

Reply:
Hmm 🤔 Even though we didn't technically process a message?

Reply:
https://github.com/nasa/cumulus/pull/3389/commits/d7a901399c8c025aa0238aff89a01d01cf98975b @jennyhliu updated to return 1 when message is deleted